### PR TITLE
Feature/components/update tag icon

### PR DESCRIPTION
**Pull request by akinogu**
>Wrong hover state
Correct: The tag with a close button should have separate hover and click states for the tag itself and the 'X', just like with segmented dropdown.

### CODE REVIEW

File: src/style/components/_tags.scss [hunk 1]
```diff
@@ -31,21 +22,60 @@ $label-font: 12px;
   &.rounded {
     border-radius: 12px;
   }
-  &.icon {
-    padding-right: 20px;
-    .content {
-      cursor: pointer;
+}
+$tag-close-width: 21px;
```
Comment (syug) [inline src/style/components/_tags.scss]: Updated the width

File: src/style/components/_tags.scss [hunk 2]
```diff
@@ -31,21 +22,60 @@ $label-font: 12px;
...
+.tag-wrapper.icon {
+  display: inline-block;
+  padding-right: $tag-close-width;
+  position: relative;
+  line-height: 1.2;
+  font-size: $label-font;
+  .tag {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+    padding-right: 3px;
```
Comment (syug) [inline src/style/components/_tags.scss]: Updated the padding

**Comment by syug** [comment]
Could you please add square version of Tag & Icon?

**Comment by akinogu** [comment]
>Could you please add square version of Tag & Icon?

I added square version of Tag & Icon.

**Pull request merged**